.vib/minio/cypress/cypress/integration/minio_test_suite.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.get('[data-testid="ErrorOutlineIcon"]').should('not.exist');
+  cy.get("#logout").scrollIntoView().should('be.visible').click();
+})
+
+it('allows creating a bucket and file upload', () => {  
+  cy.login();
+  cy.visit('add-bucket');
+  cy.fixture('testdata').then((td) => {
+    cy.get('#bucket-name').should('be.visible').type(`${td.bucketTitle}.${random}`);
```

Comment:
As a general rule of thumb, I always assert the content of the field I have changed with `type` has been updated with `should('have.text/have.value`. Do you think it is someone we should implement?

Reply:
As discussed in another conversation, and consulted with [Cypress documentation](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Asserting-in-English), we can omit this assertion since it's not really probable a failure in typing in will happen. Cypress already asserts the type-ability of an element before trying to type into it.